script/restart-services.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# TODO
+# error checking on each command
+# how to pass ssh key to github actions
+
+set -e
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+
+if [ $1 == 0 ]
+  then
+    echo -e "${RED}provide an ssh key as an argument${NC}"
+    exit 1
+fi
+
+echo -e "${GREEN}Fetching latest master${NC}"
+cd OSUMC-Cultural-Awareness-App/
+git fetch
+git pull origin master
+
+echo -e "${GREEN}Installing python packages${NC}"
+/home/ec2-user/.local/bin/pipenv install
```

Comment:
If we're using pipenv can we remove `requirements.txt`?

Reply:
yeah I thought I already got rid of it